libmscore/instrument.cpp

```diff
@@ -323,7 +323,7 @@ bool Instrument::readProperties(XmlReader& e, Part* part, bool* customDrumset)
             // be created
             if (!_drumset)
                   _drumset = new Drumset(*smDrumset);
-            if (!customDrumset) {
+            if (!*customDrumset) {
```

Comment:
Is this change causing the Travis CI failure? Why? If not, what else?

Reply:
It's this line. The test is wrong. I will merge manually and fix the test.